pkg/vadmin/revive_db_vc.go

```diff
@@ -67,5 +67,12 @@ func (v *VClusterOps) genReviveDBOptions(s *revivedb.Parms, certs *HTTPSCerts) *
 	opts.Cert = certs.Cert
 	opts.CaCert = certs.CaCert
 
+	// restore options
```

Comment:
If the restore point is set for the admintools revive, we should return an error.

Reply:
I thought we already did this at the beginning of the revive db reconcile loop?

Reply:
Okay, that should be sufficient. Please disregard this then.